source/standards/operating-systems.html.md.erb

```diff
@@ -1,34 +1,18 @@
 ---
-title: Operating Systems
+title: How to choose operating systems
 expires: 2018-01-31
```

Comment:
Does this date need moving on now?

Reply:
Not at the moment, expiry dates were\are agreed by the tech forum. So this is just to tighten up language and ensure any content we publish externally matches GOV.UK style.